src/modules/orders/dtos/notification.dto.ts

```
@@ -0,0 +1,47 @@
+import { IsNotEmpty, IsNumber,IsEmail, IsString } from 'class-validator';
```

ESLint

[error] 'IsNumber' is defined but never used.

(@typescript-eslint/no-unused-vars)